apps/web/src/app/[locale]/tracks/_components/track-challenge-card.tsx

```diff
@@ -47,18 +59,8 @@ export function TrackChallenge({ challenge, className, mock }: TrackChallengePro
         )}
       >
         <div className="relative flex items-center gap-3 text-xs sm:text-base">
-          {mock == true && (
-            <>
-              <input
-                className="peer hidden appearance-none"
-                type="checkbox"
-                id={challenge.id.toString()}
-              />
-              <div className="h-5 w-5 rounded-full border border-black/70 bg-black/10 duration-75 peer-checked:border-transparent peer-checked:bg-green-600/80 dark:border-white/50 dark:bg-white/10 peer-checked:dark:bg-green-300/80" />
-              <Check className="absolute left-1 my-auto h-3 w-3 scale-0 stroke-[4] text-white duration-300 peer-checked:scale-100 dark:text-black" />
-            </>
-          )}
-          {challenge.name}
+          <MockComponent challenge={challenge} />
+          <div>{challenge.name}</div>
```

Comment:
I don't think we need this extra div here

Reply:
I'll remove it